packages/server-admin-ui/src/views/Dashboard/Dashboard.js

```diff
@@ -30,6 +32,13 @@ const Dashboard = props => {
             <CardBody>
               <Row>
                 <Col xs='12' md='6'>
+                  <div className='callout callout-primary'>
+                    <small className='text-muted'>
+                      Signal K Uptime
```

Comment:
How about _Server Uptime_, _Signal K Server Uptime_ or just _Uptime_?

Reply:
I changed to `Uptime`